FEM/fem_ele_vec.cpp

```diff
@@ -1325,6 +1298,9 @@ void CFiniteElementVec::GlobalAssembly_Stiffness()
 	if (act_nodes_h != nnodesHQ)
 		non_ghost_flag = -1;
 
+	petsc_group::PETScLinearSolver* eqs = pcs->eqs_new;
+
+	const PetscInt zero_id_alias = -eqs->Size();
```

Comment:
why we cannot use  `-1` instead of `-eqs->Size()`? 

> Actually, the bug existed in this function before.
Would you give us bit more explanation about this?